h2o-algos/src/main/java/hex/schemas/GLMV3.java

```diff
@@ -152,31 +166,38 @@ public static final class GLMParametersV3 extends ModelParametersSchemaV3<GLMPar
     @API(help = "Perform scoring for every score_iteration_interval iterations", level = Level.secondary)
     public int score_iteration_interval;
 
-    @API(help = "Standardize numeric columns to have zero mean and unit variance", level = Level.critical, gridable = true)
+    @API(help = "Standardize numeric columns to have zero mean and unit variance", level = Level.critical,
+            gridable = true)
     public boolean standardize;
 
     @API(help = "Only applicable to multiple alpha/lambda values.  If false, build the next model for next set of " +
             "alpha/lambda values starting from the values provided by current model.  If true will start GLM model " +
             "from scratch.", level = Level.critical)
     public boolean cold_start;
 
-    @API(help = "Handling of missing values. Either MeanImputation, Skip or PlugValues.", values = { "MeanImputation", "Skip", "PlugValues" }, level = API.Level.expert, direction=API.Direction.INOUT, gridable = true)
+    @API(help = "Handling of missing values. Either MeanImputation, Skip or PlugValues.", 
+            values = { "MeanImputation", "Skip", "PlugValues" }, level = API.Level.expert, 
+            direction=API.Direction.INOUT, gridable = true)
     public GLMParameters.MissingValuesHandling missing_values_handling;
     
     @API(help = "If set to dfbetas will calculate the difference in beta when a datarow is included and excluded in " +
             "the dataset.", values = { "dfbetas" }, level = API.Level.expert, gridable = false)
     public GLMParameters.Influence influence;
 
-    @API(help = "Plug Values (a single row frame containing values that will be used to impute missing values of the training/validation frame, use with conjunction missing_values_handling = PlugValues)", direction = API.Direction.INPUT)
+    @API(help = "Plug Values (a single row frame containing values that will be used to impute missing values of the" +
+            " training/validation frame, use with conjunction missing_values_handling = PlugValues)", 
+            direction = API.Direction.INPUT)
     public FrameKeyV3 plug_values;
     
     @API(help = "Restrict coefficients (not intercept) to be non-negative")
     public boolean non_negative;
 
-    @API(help = "Maximum number of iterations", level = Level.secondary)
+    @API(help = "Maximum number of iterations.  Value should >=1.  A value of 0 is only set when only the model " +
+            "coefficient names and model coefficient dimensions are needed.", level = Level.secondary)
     public int max_iterations;
 
-    @API(help = "Converge if  beta changes less (using L-infinity norm) than beta esilon, ONLY applies to IRLSM solver ", level = Level.expert)
+    @API(help = "Converge if  beta changes less (using L-infinity norm) than beta esilon, ONLY applies to IRLSM solver "
```

Comment:
```suggestion
    @API(help = "Converge if beta changes less (using L-infinity norm) than beta esilon, ONLY applies to IRLSM solver "
```

Reply:
I like to keep the expert level label because it is actually affects the optimization.

Reply:
Changed.